python/cuml/cuml/internals/interop.py

```diff
@@ -58,6 +58,17 @@ class UnsupportedOnCPU(ValueError):
     """An exception raised when a conversion of a GPU to a CPU estimator isn't supported"""
 
 
+def is_fitted(model) -> bool:
```

Comment:
Can we just use `sklearn.utils.validation.check_is_fitted()` instead of this custom function?

Reply:
Not yet (and maybe not worth it anyway).

- `check_is_fitted` raises an error, rather than a boolean check. We could _wrap_ `check_is_fitted` to catch the error and return a bool, but that seems a bit convoluted.
- As is, most cuml estimators don't work correctly with `check_is_fitted` since they have fitted attributes initialized to `None` or use `property` to define them. We should fix this, but that will come later as we cleanup our own estimators to better match sklearn conventions.

Defining our own utility avoids both these issues, and seems the best decision. It's also not that many lines of code.